3-enrich/scala-hadoop-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.hadoop/good/CljTomcatSendgridEventSpec.scala

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2012-2014 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich
+package hadoop
+package good
+
+// Scala
+import scala.collection.mutable.Buffer
+
+// Specs2
+import org.specs2.mutable.Specification
+
+// Scalding
+import com.twitter.scalding._
+
+// Cascading
+import cascading.tuple.TupleEntry
+
+// This project
+import JobSpecHelpers._
+
+/**
+  * Holds the input and expected data
+  * for the test.
+  */
+object CljTomcatSendgridEventSpec {
+
+  val lines = Lines(
+    "2014-10-09  16:28:31    -   13  255.255.255.255   POST    255.255.255.255   /com.sendgrid/v3   404 -  -    aid=email&cv=clj-0.6.0-tom-0.0.4&nuid=-   -   -   -   application%2Fjson   W3siZW1haWwiOiJleGFtcGxlQHRlc3QuY29tIiwidGltZXN0YW1wIjoxNDQ2NTQ5NjE1LCJzbXRwLWlkIjoiXHUwMDNjMTRjNWQ3NWNlOTMuZGZkLjY0YjQ2OUBpc210cGQtNTU1XHUwMDNlIiwiZXZlbnQiOiJwcm9jZXNzZWQiLCJjYXRlZ29yeSI6ImNhdCBmYWN0cyIsInNnX2V2ZW50X2lkIjoic1pST3dNR01hZ0Znbk9FbVNkdmhpZz09Iiwic2dfbWVzc2FnZV9pZCI6IjE0YzVkNzVjZTkzLmRmZC42NGI0NjkuZmlsdGVyMDAwMS4xNjY0OC41NTE1RTBCODguMCJ9XQ=="
```

Comment:
Is `aid=email&cv=clj-0.6.0-tom-0.0.4&nuid=` a set of get params? Sendgrid doesn't appear to send any of these, but they should be captured still